httpbp/example_server_test.go

```diff
@@ -94,9 +93,9 @@ var (
 func ExampleNewBaseplateServer() {
 	var cfg config
 	ctx, bp, err := baseplate.New(context.Background(), baseplate.NewArgs{
-		EdgeContextImpl: ecinterface.Mock(), // In real code this should be a real implementation instead.
-		ConfigPath:      "example.yaml",
-		ServiceCfg:      &cfg,
+		ConfigPath:         "example.yaml",
+		EdgeContextFactory: nil, // In real code this should be the factory from the actual implementation.
```

Comment:
Note, I think having this be nil will cause `baseplate.New` to panic, so probably not what we want to do in an example 😅

Reply:
I don't feel like providing a mocked factory, though 😅 the only place the mocked factory will be used is in example code.

Reply:
Is it better now?